src/addrman.h

```diff
@@ -302,7 +349,7 @@ class CAddrMan
     {
         LOCK(cs);
 
-        unsigned char nVersion = 1;
+        unsigned char nVersion = 2;
```

Comment:
Could add code documentation here? Is this the same as line 320 above: `version byte (currently 1)`?